src/dreyfus_index.erl

```diff
@@ -118,6 +119,16 @@ init({DbName, Index}) ->
             proc_lib:init_ack(Error)
     end.
 
+maybe_create_local_purge_doc(Db, Index) ->
+    DocId = dreyfus_util:get_local_purge_doc_id(Index#index.sig),
+    case couch_db:open_doc(Db, DocId) of
+        {not_found, _} ->
+            DocContent = dreyfus_util:get_local_purge_doc_body(Db, DocId, 0, Index),
```

Comment:
I think if the local doc is not exist the purge seq is 0.
But I will modify it and the code will be more strong.
Thank you.